examples/logical_interconnects.tf

```diff
@@ -1,37 +0,0 @@
-provider "oneview" {
```

Comment:
why did you delete this..? Is it by mistake..?

Reply:
2 example files of LI was unnecessarily created, in incorporated to one. check LI example file.